proto/osmosis/bridge/v1beta1/bridge.proto

```diff
@@ -11,10 +11,41 @@ message Params {
   repeated string signers = 1 [ (gogoproto.moretags) = "yaml:\"signers\"" ];
   // Assets is a list used to create tokenfactory denoms
   // for corresponding trading pairs
-  repeated AssetWithStatus assets = 2 [
+  repeated Asset assets = 2 [
     (gogoproto.moretags) = "yaml:\"assets\"",
     (gogoproto.nullable) = false
   ];
+  // VotedNeeded marks how many signers out of the list of signers need
+  // to sign until a tx can be considered finalized
+  uint64 votes_needed = 3 [ (gogoproto.moretags) = "yaml:\"votes_needed\"" ];
+  // Fee defines a param for fee that go towards the validator set
+  // signing the incoming/outgoing txs. The fee is measured as a ratio,
+  // so its value lies between 0 and 1.
+  string fee = 4 [
+    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
+    (gogoproto.moretags) = "yaml:\"fee\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+// AssetID defines a pair of the source chain name and its Osmosis
+// representation denoted by denom. AssetID is a primary key for Asset.
+message AssetID {
+  // SourceChain is a source chain name
+  string source_chain = 1 [ (gogoproto.moretags) = "yaml:\"source_chain\"" ];
+  // Denom is the Osmosis representation of the SourceChain
+  string denom = 2 [ (gogoproto.moretags) = "yaml:\"denom\"" ];
+}
+
+// Asset is a representation of the asset.
+message Asset {
+  // ID is the asset's primary key
+  AssetID id = 1
+      [ (gogoproto.moretags) = "yaml:\"id\"", (gogoproto.nullable) = false ];
+  // Status is a current status of the asset
```

Comment:
```suggestion
  // AssetStatus is a current status of the asset
```

Reply:
Why so? `Status` already satisfies godoc. The comment should start with the parameter name (in camel case), and the parameter name is `status`